modules/cloud-router-connection/main.tf

```diff
@@ -168,4 +184,21 @@ resource "equinix_fabric_connection" "secondary_cloud_router_connection" {
       }
     }
   }
+
+  dynamic "z_side" {
+    for_each = var.zside_ap_type == "VD" ? [1] : []
+    content {
+      access_point {
+        type = "VD"
+        virtual_device {
+          type = var.zside_vd_type
+          uuid = var.zside_vd_uuid
```

Comment:
We're not letting the customer choose two different VDs with this implementation. I think it's better if we allow the customer to use a different secondary VD. This may be why you got the error message that you could only have 1 FCR for both connections because you were using the same VD on the zside, just different interfaces.

You're also not using the secondary interface value here. I think the error you saw was because of incorrect setup in the module. Not because of a product restriction.

You should try for 2 FCR to 2 VD again. To allow the customer the options to choose.

Reply:
I have tried using 2 FCR to 2 VD again. It's working correctly, I forgot to change the secondary variables values in cloud-router-module. I have updated the code.